src/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProvider.php

```diff
@@ -252,4 +253,31 @@ private function findModuleCategory(ApiModule $installedProduct, array $categori
 
         return CategoriesProvider::CATEGORY_OTHER;
     }
+
+    /**
+     * Sort addons categories by order field
+     *
+     * @param array $categories
+     */
+    private function sortCategories(array $categories)
+    {
+        uasort(
+            $categories,
+            function ($a, $b) {
+                $a = !isset($a->order) ? 0 : $a->order;
+                $b = !isset($b->order) ? 0 : $b->order;
+
+                if ($a === $b) {
+                    return 0;
+                }
+
+                return ($a < $b) ? -1 : 1;
+            }
+        );
+
+        // Convert array to object to be consistent with current API call
+        $categories = json_decode(json_encode($categories));
```

Comment:
😄

Reply:
Don't want to introduce a BC break if I change the return of the API to an array :(

Reply:
https://media.giphy.com/media/c6DIpCp1922KQ/giphy.gif